File: 1_week/stack.go
package main

import "fmt"

func main(){
  var num int

  fmt.Print("Enter a number: ")
  fmt.Scan(&num)

  for num < 0{
    fmt.Print("Enter a number: ")
    fmt.Scan(&num)
  }

  for x := 0; x < num ; x++{
    for y := 0; y < num; y++{
      fmt.Print("# ")
    }
    fmt.Print("\n")
  }
}

File: 1_week/stack.c
#include <stdio.h>

void main(void) {

  int num;

  printf("Please enter a number: ");
  scanf("%i", &num);

  if(num < 0){
    printf("Please enter a number: ");
    scanf("%i", &num);
  }

  for(int x = 0; x < num; x++){
    for(int x = 0; x < num; x++){
      printf("# ");
    }
    printf("\n");
  }
}

